### ISSUE TITLE
performance cuda/roc doc page

### ISSUE THREAD

**Issue by cristipurdel** [numba/numba#3698]
## Feature request

It would be helpful to have a page for cuda/roc where the maximum performance can be achieved.
From my experience, @vectorize is the fastest decorator but cuda has some limitations on the functions it uses, see http://numba.pydata.org/numba-doc/latest/cuda/cudapysupported.html
What I would like is to have a dedicated performance page where at least there are links to:
- which data types are working everytime (since what does not work for vectorize might work for cuda.jit)
- which functions can be used

Basically all functions / data types which must work with all architectures (cuda + roc) for the latest stable version. This will give a better overview of what is working and even what might be implemented in future.

This will make it easier for me to know if a solution for #3164 has been implemented

**Comment by stuartarchibald** [comment]
Thanks for opening this. Since this ticket was written, support for ROCm has been removed from Numba as it was essentially unmaintained (removed in version 0.54.0). Am going to close this as a result. Thanks again!
